apps/org/views.py

```diff
@@ -56,3 +66,213 @@ def get_queryset(self):
         """A user may only delete Organizations that they are associated with."""
         qs = super().get_queryset()
         return qs.filter(users=self.request.user)
+
+
+@login_required(login_url='home')
+def org_create_member_view(request, org_slug):
+    """A view for allowing a User at an Organization to create a Member for that Organization."""
+    organization = get_object_or_404(
+        Organization.objects.filter(users=request.user),
+        slug=org_slug
+    )
+
+    # If the user is GETting the page, then render the template.
+    if request.method == 'GET':
+        context = {'organization': organization, 'form': CreateNewMemberAtOrgForm()}
+        return render(request, 'org/org_create_member.html', context=context)
+    else:
+        # The user is POSTing to create a new Member at the organization, so we:
+        #   1.) validate the input
+        #   2.) make a request to VMI to create the new user
+        #   3.) create a new Member with the response from VMI
+
+        # 1.) Validate the input. If the form is not valid, then return the errors
+        # to the user.
+        form = CreateNewMemberAtOrgForm(request.POST)
+        if not form.is_valid():
+            return render(
+                request,
+                'org/org_create_member.html',
+                context={'organization': organization, 'form': form}
+            )
+
+        # 2.) Make a request to VMI to create the new user
+        request_user_social_auth = request.user.social_auth.filter(
+            provider=settings.SOCIAL_AUTH_NAME
+        ).first()
+        # If the request.user does not have a UserSocialAuth for VMI, then
+        # return the error to the user.
+        if not request_user_social_auth:
+            errors = {'user': 'User has no association with {}'.format(settings.SOCIAL_AUTH_NAME)}
+            return render(
+                request,
+                'org/org_create_member.html',
+                context={'organization': organization, 'form': form, 'errors': errors}
+            )
+        # The data to be POSTed to VMI
+        data = {
```

Comment:
could move this vmi communication logic into a form class, allowing a pattern like [this](https://docs.djangoproject.com/en/2.2/ref/class-based-views/generic-editing/#formview)

```python
from myapp.forms import ContactForm
from django.views.generic.edit import FormView

class ContactView(FormView):
    template_name = 'contact.html'
    form_class = ContactForm
    success_url = '/thanks/'

    def form_valid(self, form):
        # This method is called when valid form data has been POSTed.
        # It should return an HttpResponse.
        form.send_email()
        return super().form_valid(form)
```